examples/web/src/jobs/@tokens.scss

```diff
@@ -17,17 +17,21 @@
     $emotion-warning-default: #ff9900,
     $emotion-warning-background: #fff5e6,
     $emotion-danger-default: #d40014,
-    $emotion-danger-background: #fbe6e8,
+    $emotion-danger-background: #fbe6e8
 );
 @forward '@lmc-eu/spirit-design-tokens/dist/scss/radii' with (
     $radius-1: 0,
     $radius-2: 0,
-    $radius-3: 0,
+    $radius-3: 0
 );
 @forward '@lmc-eu/spirit-design-tokens/dist/scss/shadows';
 @forward '@lmc-eu/spirit-design-tokens/dist/scss/spacing';
 @forward '@lmc-eu/spirit-design-tokens/dist/scss/typography' with (
-    $font-family-default: #{'Open Sans', helvetica, roboto, arial, sans-serif},
-    $font-size-3: 13px,
-    $font-size-4: 15px,
-);
+        $font-family-default: #{'Open Sans',
```

Comment:
Do we have to do this?

Reply:
This is [done by Prettier](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEABAZhATgdwIZYAmABAOSoA2AtmALRwCuA9AM4AOAllhzLYXCw4BzKLRgQA1ghZNCHFjFZgWMmAE82EIVjxsAFmtLEcPPcQAUAHSjFbd2wBJMsWujxUOFNXzhuGFGCRiAGJgUgB5NgRiAGU8KBZSABpiPTgKADc4GA4wPBSsCAAjCHEUgg48ChSWeJZaFjhudABfJOt7eydoXkEALzhaAGYggEYhtgAPdptOx2dejgHaABYxgFYpjrsASgBuECSQCDYc6BZkUAJCnAAFAmlkECr8NQujop0wKRg4qjgADIcKBwZBuCiND5fH4xNh4MDAoTIGBYBhwI5wKhFOCEfiEAHxIQMPBCOAAMWwVDwMByUCRKDwDHEhxAehgVAoAHU9DwBHCwHAYtIeBwMjw1E9lO8QMDGlgYLcdEIqWCqpCQAArFiTGKIihwACKDFKoKQ4PVcKwcqeLClLLY3FgnI4hBgemQowATAAGI4OiCNTk6NhPB0CJpZFkAR2N8EVJwuDPqIJxOJZWDgMa4cEVJJVZrV6JAjQ8yNRRcEdP1RpNqohRZgeCKztd7qQnqOKLwnkRAGEIFR8yABOsWQxGgAVJuJ81FjJogCSUH4sBiYG4pwAgsuYup9XXGi0WkA).

Reply:
Maybe this should be formated by stylelint only with the prettier plugin. And ignore it with prettier itself, maybe.